### DISCUSSION TITLE
Add album art to FLAC files using Audacious

### DISCUSSION THREAD

**Discussion by ScubaMan1970** [audacious-media-player/audacious#1463] [General]
**I've been using Mint 21.3** since the end of June, 2024. I had to give up my favourite WinOS disc ripping software, Exact Audio Copy (_EAC_). And give up on Foobar2000. I'm very pleased with Audacious 4.4 (_thank you again updating it_). 
[I used this site to update my version to Audacious.](https://thelinuxcode.com/audacious-advanced-audio-player/)

I used **Asunder CD ripper** to make FLAC copies of my **purchased** music discs. **Question:** Can Audacious 4.4 add album art to the music files? I have the album art as JPG files.

**Comment by ScubaMan1970** [comment]
**SOLVED.** I'm using a designated program to edit music file meta-data. I'm having luck with Puddletag audio tag editor. It's an older program... 2014. I'll see how it goes. If YOU have issues, consider trying it out: link = https://docs.puddletag.net/source/tut3.html#adding-album-art

**Comment by Npa972** [comment]
Hi give a try to kid3-qt. It's up to date and very good for tagging, embed cover...
If you use audacious with qt6 then kid3-qt will not add a lot of dependencies.

**Reply by mschwendt** [reply]
EasyTAG is popular and widely available, too. And a powerful GUI when it comes to editing tags in a high number files.

**Reply by ScubaMan1970** [reply]
**SOLVED & SUGGESTIONS APPRECIATED.** EasyTAG and kid3-qt. Interesting! I'll check it out.
